js/file.js

```diff
@@ -91,3 +91,50 @@ const authorInput = document.querySelector('#author');
 formAdd.addEventListener('click', () => {
   books.addBook(titleInput.value, authorInput.value);
 });
+
+const formSection = document.querySelector('#add');
+const showSection = document.querySelector('#show');
+const contactSection = document.querySelector('#contact');
+const listLink = document.querySelector('#listLink');
+const addLink = document.querySelector('#addLink');
+const contactLink = document.querySelector('#contactLink');
+
+const navButtons = document.querySelectorAll('.nav-link');
+
+function addNew() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.